pats_service.py

```diff
@@ -0,0 +1,653 @@
+                                "trapeye_level_3": 10,
+                                "trapeye_level_4": 15
+                            },
+                        ],
+                        "greenhouse_name": "your_greenhouse_name",
+                        "hubspot_company_id": 0123456789,
+                        "id": 123,
+                        "label": "section_label",
+                        "n_weekly_trapeye_photos": 3,
+                        "name": "a_name",
+                        "timezone": "Europe/Amsterdam"
+                    }
+                ]
+            }
+        }
+
+        Returns:
+            dict: dict containing all sections. Each section contains meta data about it self,
+                  and a list of detection classes available to that section.
+        """
+        self.logger.debug("Retrieving sections from pats server")
+
+        # Initialize headers.
+        headers = {'Authorization': 'Bearer ' + self.token}
+
+        # Send request, and validate response code.
+        response = requests.get(self.server + '/api/sections', headers=headers, timeout=self.timeout)
+        if response.status_code != 200:
+            self.logger.critical(f"Download sections failed: {str(response.status_code)}, msg: {response.text}")
+            sys.exit(1)
+
+        self.logger.info("Successfully sections from pats server")
+        return response.json()['sections']
+
+    def download_spots(self, section_id: int, map_snapping: bool = True) -> dict:
+        """Method used to download the spots from the Pats server.
+        The json consists of two dictionaries, the first containing the spots for c systems,
+        and the second containing spots for trapeye systems.
+
+        json example:
+            {
+                "c": [
+                    {
+                        "label": "a_label",
+                        "latitude": 12.3456789,
+                        "longitude": 9.8765432,
+                        "post_id": 21,
+                        "row_id": 42,
+                        "system_id": 123
+                    },
+                ],
+                 "trapeye": [
+                    {
+                        "latitude": 98.76543210987654,
+                        "longitude": 1.2345678901234,
+                        "post_id": 42,
+                        "row_id": 21,
+                        "unit_id": 9876
+                    },
+                ]
+            }
+
+        Args:
+            section_id (int, optional): the id of the section for which the spots will be downloaded.
+            map_snapping(bool, optional): flag to turn on map snapping on the hand placed location of sensors. Defaults to True.
+
+        Returns:
+            dict: a dict containing the json response body.
+        """
+        self.logger.debug("Downloading spots from pats server")
+
+        # Convert boolean to int.
+        map_snapping_num: int = self.__convert_bool_to_int(map_snapping)
```

Comment:
It is also possible to convert a bool to an int by casting it to an int
```suggestion
        map_snapping_num: int = int(map_snapping)
```